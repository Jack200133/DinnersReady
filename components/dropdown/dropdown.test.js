import React from 'react';
import renderer from 'react-test-renderer';
import {render, screen} from '@testing-library/react-native';
import Dropdown from './dropdown';

describe('<Dropdown />', () => {
  it('renders the correct options', () => {
    /*const tree = renderer.create(<Dropdown />).toJSON();
    expect(tree).toMatchSnapshot();*/
  })
})