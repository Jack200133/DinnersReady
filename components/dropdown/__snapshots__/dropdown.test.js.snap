// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Dropdown /> renders the correct options 1`] = `
<View>
  <View
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
  >
    <View
      style={
        Object {
          "backgroundColor": "#EEE9E9",
          "borderRadius": 18,
          "paddingHorizontal": 15,
          "paddingVertical": 10,
          "width": "100%",
        }
      }
    >
      <Text
        style={
          Object {
            "fontSize": 18,
          }
        }
      >
        Elegir dificultad
      </Text>
    </View>
  </View>
</View>
`;
